files/en-us/mozilla/firefox/releases/12/index.html

```diff
@@ -69,7 +69,7 @@ <h3 id="SVG">SVG</h3>
 <h3 id="MathML">MathML</h3>
 
 <ul>
- <li>To control the directionality of MathML formulas, the <code>dir</code> attribute is now supported on the {{MathMLElement("math")}}, {{MathMLElement("mrow")}}, and {{MathMLElement("mstyle")}} elements as well as on <a href="/Special:Tags?tag=MathML:Token+Elements" title="Special:Tags?tag=MathML:Token+Elements">MathML Token Elements</a>. This is particularly important for some <a class="external" href="https://www.w3.org/TR/arabic-math/">Arabic mathematical notations</a>.</li>
+ <li>To control the directionality of MathML formulas, the <code>dir</code> attribute is now supported on the {{MathMLElement("math")}}, {{MathMLElement("mrow")}}, and {{MathMLElement("mstyle")}} elements as well as on <a href="https://developer.mozilla.org/en-US/docs/Web/MathML/Element#token_elements">MathML Token Elements</a>. This is particularly important for some <a class="external" href="https://www.w3.org/TR/arabic-math/">Arabic mathematical notations</a>.</li>
```

Comment:
`flaws` will re-strip the FQDN
```suggestion
 <li>To control the directionality of MathML formulas, the <code>dir</code> attribute is now supported on the {{MathMLElement("math")}}, {{MathMLElement("mrow")}}, and {{MathMLElement("mstyle")}} elements as well as on <a href="/en-US/docs/Web/MathML/Element#token_elements">MathML Token Elements</a>. This is particularly important for some <a class="external" href="https://www.w3.org/TR/arabic-math/">Arabic mathematical notations</a>.</li>
```

Reply:
Thanks - sorry about that :-(